mln/models/static.py

```diff
@@ -324,3 +306,47 @@ def __str__(self):
 class StartingStack(Stack):
 	"""A stack that users start off with in their inventory when they create an account."""
 	item = models.OneToOneField(ItemInfo, related_name="+", on_delete=models.CASCADE)
+
+class MessageTemplate(models.Model): 
+	networker = models.ForeignKey(User, related_name="+", on_delete=models.CASCADE, blank=True, null=True, limit_choices_to={"profile__is_networker": True})
+	body = models.ForeignKey(MessageBody, related_name="+", on_delete=models.CASCADE)
+
+	def __str__(self): 
+		return "From %s: %s" % (self.networker, self.body)
+
+class MessageTemplateAttachment(Stack): 
+	template = models.ForeignKey(MessageTemplate, related_name="attachments", on_delete=models.CASCADE)
+
+	class Meta: 
+		constraints = (models.UniqueConstraint(fields=("template", "item"), name="messsage_template_attachment_unique_template_item"),)
+
+class NetworkerReply(models.Model): 
+	template = models.ForeignKey(MessageTemplate, related_name="+", on_delete=models.CASCADE)
+	message_body = models.ForeignKey(MessageBody, related_name="+", on_delete=models.CASCADE, null=True, blank=True)
+	message_attachment = models.ForeignKey(ItemInfo, related_name="+", on_delete=models.CASCADE, null=True, blank=True)

```

Comment:
I think we should make it clearer what's the reply here and what's being replied to. `template` -> `reply` seems straightforward, I'm not so sure about `message_body` and `message_attachment`. Google suggests `prompt`, maybe you also have an idea yourself.